### ISSUE TITLE
Function prefix

### ISSUE THREAD

**Issue by rgilman** [ContextInstitute/bfcom#26]
To avoid collisions with other functions we should adopt a consistent prefix for all of our function names. I expect that "bfc-" would work. 

Thoughts on this?

**Comment by iangilman** [comment]
Good idea. Of course it should be `bfc_` not `bfc-`, but yes, that seems perfect; nice and short but distinctive.

I suppose we could consider adding `bfc-` to CSS class names for new styles we add as well, though that might get tiresome, since there may be quite a few. I think I'm in favor of doing the prefix for PHP functions but not CSS classes.
